Readme.md

```diff
@@ -57,6 +57,12 @@ https://github.com/qbittorrent/qBittorrent/issues
     * `QBT_WEBUI_PORT` \
       This environment variable sets the port number which qBittorrent WebUI will be binded to.
       Defaults to port `8080` if value is not set.
+    * `QBT_IGNORE_PERMISSIONS` \
+      This environment variable skips all the permission preflight checks and
+      doesn't try to do anything with the users. Set to `1` **only if you
+      understand what and why you're doing**. This allows qBittorrent-nox to run
+      in the environments where the user id is enforced outside of the
+      container.
```

Comment:
"the user id is enforced outside of the container" what does it mean? Could you elaborate?

Reply:
e.g. with a following security context in kubernetes:

```
securityContext:
  runAsUser: 1000
  runAsGroup: 1000
  allowPrivilegeEscalation: false
  readOnlyRootFilesystem: true
```

qBitTorrent will be started as 1000:1000, and you won't be able to call doas, or, most probably, chown the files.

Reply:
>qBitTorrent will be started as 1000:1000, and you won't be able to call doas, or, most probably, chown the files.


I see. But As you can see, it is supposed to be the container job to lower the privileges (run as non-root user) or at least this is the designated usage.
Docker also has similar directives: https://docs.docker.com/reference/dockerfile/#user but it is not supposed to be used when using this image. The readme/manual has given exact steps of how to run it.

So you shouldn't use these security contexts in kubernetes when running this image, or are you not able to disable them?